addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -378,6 +378,73 @@ QUnit.test('open chat from "new message" chat window should open chat in place o
     );
 });
 
+QUnit.test('new message chat window should close on selecting the user if chat with the user is already open', async function (assert) {
+    assert.expect(4);
+
+    this.data['res.partner'].records.push({ id: 131, name: "Partner 131"});
+    this.data['res.users'].records.push({ id: 12, partner_id: 131 });
+    this.data['mail.channel'].records.push({
+        id: 20,
+        is_minimized: true,
```

Comment:
this is the default value, but if you want to explicit it the initial state which is fine, also add `state: 'open'`